src/page/address-step.page.ts

```diff
@@ -4,7 +4,11 @@ export class AddresStepPage {
   private proceedCheckoutButton: ElementFinder;
 
   constructor () {
-    this.proceedCheckoutButton = $('#center_column > form > p > button > span');
+    /* It is not very clear what element is being selected and can easily crash if other elements
+    are added or the order of them changes. We fix it the same as the past ones: tying to be more
+    specific, based on id's, names and properties about what exact element we want to select*/
+    this.proceedCheckoutButton = $(
+      'form > p.cart_navigation > button[name="processAddress"] > span');
```

Comment:
maybe `form` and `span` is not needed

Reply:
> maybe `form` and `span` is not needed

Yeah, maybe on some locator locators I put some extra elements trying to avoid the problem that I mentioned on Slack, but I will be more careful and try to reduce this one and the others that may have more elements than needed.

Respecting the comments, I know that on a "real" code that shouldn't be there jeje, but Holmes said on Slack that we could put the justifications on GitHub or as code comments, so I made it that way just to ease things and with purposes of the exercise, but I know that it is not a good idea on a real project.

Reply:
yes, my fault, but you can remove your comments on final commit after you get the approval for this PR

Reply:
I will do it.